evm/check-ownership-request.js

```diff
@@ -0,0 +1,194 @@
+'use strict';
+
+require('dotenv').config();
+
+const axios = require('axios');
+const { Command, Option } = require('commander');
+const { ethers } = require('hardhat');
+const { Contract, getDefaultProvider } = ethers;
+const { loadConfig, validateParameters, printError, getContractJSON, printInfo, printWarn, printObj } = require('./utils');
+
+const interchainTokenFactoryABI = getContractJSON('InterchainTokenFactory').abi;
+const interchainTokenABI = getContractJSON('InterchainToken').abi;
+const interchainTokenServiceABI = getContractJSON('InterchainTokenService').abi;
+const erc20ABI = getContractJSON('IERC20Named').abi;
+
+async function processCommand(config, options) {
+    try {
+        const { deployer, address, its, rpc, api } = options;
+        let { source, destination } = options;
+
+        validateParameters({ isValidAddress: { address }, isNonEmptyString: { source, destination } });
+
+        const sourceChain = config.chains[source.toLowerCase()];
+
+        if (!sourceChain) {
+            throw new Error(`Chain ${source} is not defined in the info file`);
+        }
+
+        destination = JSON.parse(destination);
+        const invalidDestinations = destination.filter((chain) => !config.chains[chain.toLowerCase()]);
```

Comment:
You can use this [function](https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/evm/utils.js#L1072) here if you want